### PR TITLE
Alert restructure #1

### PR DESCRIPTION
**Pull request by lucas-m98**
Fixes to alert issues

### CODE REVIEW

File: src/dashboard/luasrc/controller/ids-priv/ids_tab.lua [hunk 1]
```diff
@@ -40,7 +40,7 @@ module("luci.controller.ids-priv.ids_tab", package.seeall)  --notice that new_ta
         -- Write JSON file, and make it *pretty*,
         conf:close()
         conf = io.open("/etc/tinyHIPPO/config.json", 'w')
-        conf:write(string.gsub(luci.jsonc.stringify(configuration, true) .. "\n", "\\/", "/"))
+        conf:write((string.gsub(luci.jsonc.stringify(configuration, true) .. "\n", "\\/", "/")))
```
Comment (lucas-m98) [inline src/dashboard/luasrc/controller/ids-priv/ids_tab.lua:43]: Lua is interesting...

**Review by samir-elhelw76 — APPROVED** [review]

**Review by briweinstein — APPROVED** [review]

File: src/dashboard/luasrc/controller/ids-priv/ids_tab.lua (hunk 1 above)
Comment (MartinLPetrauskas) [reply src/dashboard/luasrc/controller/ids-priv/ids_tab.lua:43]: The fact that adding another set of parentheses solved this issue is mind boggling.

**Review by MartinLPetrauskas — APPROVED** [review]
Changes are good. OpenWrt package successfully compiled and did not crash. All known bugs relating to the config page appear to be resolved.

File: src/dashboard/luasrc/controller/ids-priv/ids_tab.lua (hunk 1 above)
Comment (lucas-m98) [reply src/dashboard/luasrc/controller/ids-priv/ids_tab.lua:43]: Apparently it decouples the values being returned, but intuitively it makes absolutely zero sense

**Pull request merged**